### PR TITLE
INN-2826: Document the Event "ids" array returned when sending events

### PR DESCRIPTION
**Pull request by djfarrelly**
This was only documented in our HTTP API response.

INN-2826

Tagging @jpwilliams for verifying accuracy with SDK.

### CODE REVIEW

File: pages/docs/reference/events/send.mdx [hunk 1]
```diff
@@ -111,6 +109,29 @@ To send events from within of the context of a function, use [`step.sendEvent()`
   </Col>
 </Row>
 
+### Return values
+
+The function returns a promise that resolves to an object with an array of Event IDs that were send. These events can be used to look up the event in the Inngest dashboard or via [the REST API](https://api-docs.inngest.com/docs/inngest-api/pswkqb7u3obet-get-an-event).
```
Comment (jpwilliams) [inline pages/docs/reference/events/send.mdx:114]:
```suggestion
The function returns a promise that resolves to an object with an array of Event IDs that were sent. These events can be used to look up the event in the Inngest dashboard or via [the REST API](https://api-docs.inngest.com/docs/inngest-api/pswkqb7u3obet-get-an-event).
```

File: pages/docs/reference/functions/step-send-event.mdx [hunk 2]
```diff
@@ -39,7 +39,7 @@ To send events from outside of the context of a function, use [`inngest.send()`]
 
 ---
 
-## `step.sendEvent(id, eventPayload | eventPayload[]): Promise`
+## `step.sendEvent(id, eventPayload | eventPayload[]): Promise<{ ids: String[] }>`
```
Comment (jpwilliams) [inline pages/docs/reference/functions/step-send-event.mdx:42]:
```suggestion
## `step.sendEvent(id, eventPayload | eventPayload[]): Promise<{ ids: string[] }>`
```

File: pages/docs/reference/events/send.mdx [hunk 3]
```diff
@@ -25,9 +25,7 @@ To send events from within of the context of a function, use [`step.sendEvent()`
 ---
 
 
-## `inngest.send(eventPayload | eventPayload[]): Promise`
-{/* TODO - Show the event ids in the return value when we add to the SDK */}
-{/*## `inngest.send(eventPayload | eventPayload[]): Promise<string[]>`*/}
+## `inngest.send(eventPayload | eventPayload[]): Promise<{ ids: String[] }>`
```
Comment (jpwilliams) [inline pages/docs/reference/events/send.mdx:28]:
```suggestion
## `inngest.send(eventPayload | eventPayload[]): Promise<{ ids: string[] }>`
```

File: pages/docs/events/index.mdx [hunk 4]
```diff
@@ -178,6 +178,29 @@ export default inngest.createFunction(
 );
 ```
 
+## Using Event IDs
+
+Each event sent to Inngest is assigned a unique Event ID. These `ids` are returned from `inngest.send()` or `step.sendEvent()`. Event IDs events can be used to look up the event in the Inngest dashboard or via [the REST API](https://api-docs.inngest.com/docs/inngest-api/pswkqb7u3obet-get-an-event). You can choose to log this or save these Event IDs if you want to look them up later.
```
Comment (sylwiavargas) [inline pages/docs/events/index.mdx:183]:
```suggestion
Each event sent to Inngest is assigned a unique Event ID. These `ids` are returned from `inngest.send()` or `step.sendEvent()`. Event IDs can be used to look up the event in the Inngest dashboard or via [the REST API](https://api-docs.inngest.com/docs/inngest-api/pswkqb7u3obet-get-an-event). You can choose to log or save these Event IDs if you want to look them up later.
```

**Review by jpwilliams — APPROVED** [review]
Some small tweaks - thanks for writing this!

File: pages/docs/reference/events/send.mdx (hunk 1 above)
Comment (sylwiavargas) [inline pages/docs/reference/events/send.mdx:114]:
```suggestion
The function returns a promise that resolves to an object with an array of Event IDs that were sent. These events can be used to look up the event in the Inngest dashboard or via [the REST API](https://api-docs.inngest.com/docs/inngest-api/pswkqb7u3obet-get-an-event).
```

File: pages/docs/reference/functions/step-send-event.mdx [hunk 5]
```diff
@@ -101,3 +101,26 @@ To send events from outside of the context of a function, use [`inngest.send()`]
 <Callout>
   `step.sendEvent()` must be called using `await` or some other Promise handler to ensure your function sleeps correctly.
 </Callout>
+
+### Return values
+
+The function returns a promise that resolves to an object with an array of Event IDs that were send. These events can be used to look up the event in the Inngest dashboard or via [the REST API](https://api-docs.inngest.com/docs/inngest-api/pswkqb7u3obet-get-an-event).
```
Comment (sylwiavargas) [inline pages/docs/reference/functions/step-send-event.mdx:107]:
```suggestion
The function returns a promise that resolves to an object with an array of Event IDs that were sent. These events can be used to look up the event in the Inngest dashboard or via [the REST API](https://api-docs.inngest.com/docs/inngest-api/pswkqb7u3obet-get-an-event).
```

**Review by sylwiavargas — APPROVED** [review]
Thank you! I've added a few style suggestions but it's good to go.

File: pages/docs/reference/events/send.mdx (hunk 3 above)
Comment (djfarrelly) [reply pages/docs/reference/events/send.mdx:28]: Writing too much Kotlin recently 🤦

Comment (djfarrelly) [reply pages/docs/reference/events/send.mdx:28]: Side note - this notation is kind of half JS / half TS - we probably should improve that over time

**Pull request merged**